modules/authentication/proc-enabling-authentication-with-microsoft-azure.adoc

```diff
@@ -0,0 +1,129 @@
+[id="enabling-authentication-with-microsoft-azure"]
+= Enabling authentication with Microsoft Azure
+
+{product} includes a Microsoft Azure authentication provider that can authenticate users by using OAuth.
+
+.Prerequisites
+. You have the permission to register an application in Microsoft Azure.
+. You link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/{product-version}/html-single/administration_guide_for_red_hat_developer_hub/index#assembly-add-custom-app-file-openshift_admin-rhdh[added a custom {product-short} application configuration].
+
+.Procedure
+. To allow {product-short} to authenticate with Microsoft Azure, create an OAuth Application in Microsoft Azure.
+
+.. To link: https://learn.microsoft.com/en-us/entra/identity-platform/scenario-web-app-sign-user-app-registration?tabs=aspnetcore#register-an-app-by-using-the-azure-portal[register the {product-short} application by using the Azure portal], go to link:https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationsListBlade[*Azure Portal > Identity > Identity management > App registrations*] and click **New registration**.
+
+.. On your *App registration* overview page, add a new *Web platform configuration*, with the configuration:
+
+*Redirect URI*:: Enter the backend authentication URI set in {product-short}: `pass:c,a,q[https://_<APP_FQDN>_/api/auth/microsoft/handler/frame]`
+*Front-channel logout URL*:: Leave blank.
+*Implicit grant and hybrid flows*:: Leave all checkboxes cleared.
+
+.. On the *API permissions* tab, click *Add Permission*, then add the following *Delegated permission* for the
+*Microsoft Graph API*:
++
+* `email`
+* `offline_access`
+* `openid`
+* `profile`
+* `User.Read`
+* Optional custom scopes of the Microsoft Graph API that you define both here and in the {product-short} configuration (`app-config-rhdh.yaml`).
++
+[NOTE]
+====
+Your company might require you to grant admin consent for these permissions.
+Even if your company does not require admin consent, you might do so as it means users do not need to individually consent the first time they access backstage.
+To grant admin consent, a directory admin must go to the link:https://learn.microsoft.com/en-us/azure/active-directory/manage-apps/user-admin-consent-overview[admin consent] page and click *Grant admin consent for COMPANY NAME*.
+====
+
```

Comment:
@JessicaJHee Is this step still relevant?

I cannot see this step if I follow instructions on Azure documentation in one the these docs:
* https://learn.microsoft.com/en-us/entra/identity-platform/scenario-web-app-sign-user-app-registration?tabs=aspnetcore#register-an-app-by-using-the-azure-portal
* https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app?tabs=client-secret

Reply:
Steps reviewed and updated from screencast recorded by @albarbaro.